docs: init a vuepress documentation (#2075)
ovh/manager: docs/how-to/README.md

---
sidebar: auto
---

# How To

## Install Git

> You can download the Git installer corresponding to your system [here](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git).

Reviewer: Some users might not want to use installers but package managers to install Git, it would be easier to say

```suggestion
You can find the documention to install Git there [here](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git).
```
